paper-api/src/main/java/io/papermc/paper/command/brigadier/argument/ArgumentTypes.java

```diff
@@ -291,6 +293,34 @@ public static ArgumentType<Criteria> objectiveCriteria() {
         return provider().objectiveCriteria();
     }
 
+    /**
+     * A single score holder argument. Requires resolving.
+     * 
+     * @return argument
+     */
+    public static ArgumentType<ScoreHolderResolver> scoreHolder() {
+        return provider().scoreHolder();
+    }
+
+    /**
+     * A multiple score holders argument. Requires resolving.
+     * 
+     * @return argument
+     */
+    public static ArgumentType<ScoreHolderResolver> scoreHolders() {
+        return provider().scoreHolders();
+    }
+    
+    /**
+     * An operation argument.
+     * Represents operations on scores, but can also be used for simple arithmetic.
```

Comment:
The argument type does not represent operation on scores, the Operation type does.